lua/nvim_lsp/julials.lua

```diff
@@ -0,0 +1,30 @@
+local skeleton = require 'nvim_lsp/skeleton'
+local util = require 'nvim_lsp/util'
+local lsp = vim.lsp
+
+skeleton.julials = {
+  default_config = {
+    cmd = {
+        "julia", "-e", [[ using LanguageServer; p = joinpath(dirname(dirname(pathof(LanguageServer))), "contrib", "languageserver.sh"); run(`$p`); ]]
+    };
+    filetypes = {"julia"};
+    root_dir = util.root_pattern("Project.toml");
+    log_level = lsp.protocol.MessageType.Warning;
+    settings = {};
+  };
+  -- on_new_config = function(new_config) end;
+  -- on_attach = function(client, bufnr) end;
+  docs = {
+    package_json = "https://raw.githubusercontent.com/julia-vscode/julia-vscode/master/package.json";
```

Comment:
Prefer the `vscode` key and the vscode package name if it has one, because I can scrape the package.json from there. You can see that being used in other lsp definitions.

Reply:
> Prefer the `vscode` key and the vscode package name if it has one

@kdheepak do you plan to make this change?

Reply:
I made the change in https://github.com/neovim/nvim-lsp/pull/91/commits/4e4ee285f35b65ba3ebad44259b5bb99c984c288!